app/controllers/ops_controller/settings/rhn.rb

```diff
@@ -106,12 +106,13 @@ def rhn_save_subscription
     db          = MiqDatabase.first
     credentials = rhn_credentials_from_edit
 
+    db.update_repo_name = credentials[:update_repo_name]
+
     db.update_attributes(
       credentials.slice(
         :registration_type,
         :registration_server,
-        :registration_http_proxy_server,
-        :update_repo_name
+        :registration_http_proxy_server
```

Comment:
Not sure I understand this change.  update_attributes will call the assignment, so there should be no change needed here.